webpack.config.js

```diff
@@ -18,13 +18,12 @@ const config = {
     rules: [
       {
         test: /\.js$/,
-        exclude: /node_modules/,
+        exclude: /node_modules\/(?!(prettysize)\/).*/,
```

Comment:
Why is this change necessary?

Reply:
Actually, this module is not already transpiled and Uglify is throwing an error because babel ignores it while building, any suggestion for this?

Reply:
Hm. The problem seems to be with this line:
```js
module.exports = (size, nospace, one, places) => {
```

But suppressing the Uglify warning is just masking the symptom. It will still leave an arrow function in the built bundle, which will [break IE 11 (and older)](https://caniuse.com/#feat=arrow-functions).

Taking a look at `prettisize`, the amount of work it's doing is trivial- so it's probably easier to just rewrite this logic ourselves in a util function. 😄

Reply:
Actually here babel ignores node_modules but not prettysize  so no arrow function will be in the bundle but yes I think it's not a good idea to be specific for a module. :smile:

Reply:
Ah! I misread. My apologies. 😄